### DISCUSSION TITLE
Extend 'Date From' and 'Date To' to "List of orders with taxes (PDF)"

### DISCUSSION THREAD

**Discussion by yvovandoorn** [pretix/pretix#3052] [Ideas]
Currently, the option "List of orders with taxes" includes a very handy date from to date to functionality. Very useful when the tax agency I submit to requires quarterly reports on where VAT was (not) paid. However the PDF version only does it for the entire event, no ability to sort by date. This makes the tax agency (Belastingdienst, NL) sad as they don't like XLS files. So I'm always converting each XLS tab to a PDF. 

It would be great to see the date sort exposed to the PDF version, but until then, I'll be the middle person in conversions to a format my tax agency deems "appropriate".

**Comment by raphaelm** [comment]
Yeah that makes sense and should not be to hard if someone wants to contribute it. Most if it can just be copied from the XLS version :)
